docs/node-operators/foundation-delegation-program.mdx

```diff
@@ -136,19 +130,12 @@ See the following uptime leaderboards for the latest uptime performance scores:
 
 #### Payout Addresses
 
-You must send computed rewards as described below to a Mina Foundation address - however, beginning in Cycle 9, Mina Foundation changed the method of returning rewards to the Foundation.
+You must send computed rewards as described below to a Mina Foundation address.
```

Comment:
If just sending it back to the same address that was delegated how does this differ from O(1) and Foundation? If it is indeed the same this line seems not needed and adds potential confusion?

Reply:
Oh ignore that, I see this whole page is only Foundation specific :)